cmd/azure-keyvault-env/secret_handler.go

```diff
@@ -137,7 +137,8 @@ func (h *AzureKeyVaultSecretHandler) Handle() (string, error) {
 // Handle getting and formating Azure Key Vault Certificate from Azure Key Vault to Kubernetes
 func (h *AzureKeyVaultCertificateHandler) Handle() (string, error) {
 	exportPrivateKey := h.query == corev1.TLSPrivateKeyKey
-	cert, err := h.vaultService.GetCertificate(&h.secretSpec.Spec.Vault, exportPrivateKey)
+	ensureServerFirst := h.secretSpec.Spec.Output.Secret.ChainOrder == "ensureserverfirst"
+	cert, err := h.vaultService.GetCertificate(&h.secretSpec.Spec.Vault, exportPrivateKey, ensureServerFirst)
```

Comment:
Introduce a option struct, as mentioned earlier?

Reply:
Yes, I agree, will make the changes.